modules/bullet/area_bullet.cpp

```diff
@@ -187,10 +183,11 @@ void AreaBullet::set_space(SpaceBullet *p_space) {
 	if (space) {
 		space->register_collision_object(this);
 		reload_body();
+		scratch();
 	}
 }
 
-void AreaBullet::on_collision_filters_change() {
+void AreaBullet::do_reload_collision_fm() {
 	if (space) {
```

Comment:
Nitpicking, but all these `*collision_fm` variables and functions would be better named `*collision_filters`.

Reply:
I agree, changed